lib/MusicBrainz/Server/Data/Work.pm

```diff
@@ -405,46 +416,78 @@ sub find_artists
     return %map;
 }
 
-=method load_writers
+=method load_authors
 
-This method will load the work's writers based on the work-artist
+This method will load the work's authors based on the work-artist
 relationships.
 
 =cut
 
-sub load_writers
+sub load_authors
 {
     my ($self, @works) = @_;
 
-    @works = grep { defined $_ && scalar $_->all_writers == 0 } @works;
+    @works = grep { defined $_ && scalar $_->all_authors == 0 } @works;
     my @ids = map { $_->id } @works;
     return () unless @ids;
 
     my %map;
-    $self->_find_writers(\@ids, \%map);
+    $self->_find_authors_or_misc_artists(\@ids, \%map);
     for my $work (@works) {
-        $work->add_writer(@{ $map{$work->id} })
+        $work->add_author(@{ $map{$work->id} })
             if exists $map{$work->id};
     }
 }
 
-sub _find_writers
+sub load_misc_artists
```

Comment:
Same here, it seems we call them "other artists" in the UI and "misc. artists" in the code (or maybe I'm conflating two different things).

Reply:
Hmm. I agree it would make sense to have everything use the same name. I just don't quite have a good one - but actually I don't like misc. artists because on second thought it doesn't make it clear at all it is not just "whatever" but "things other than authors and recording artists". So I might change the code to other_artists as well.